integration-tests/src/tests/nearcore/node_cluster.rs

```diff
@@ -21,7 +23,11 @@ fn start_nodes(
     epoch_length: BlockHeightDelta,
     genesis_height: BlockHeight,
     save_tx_outcomes: Option<bool>,
-) -> (Genesis, Vec<String>, Vec<(Addr<ClientActor>, Addr<ViewClientActor>, Vec<ArbiterHandle>)>) {
+) -> (
+    Genesis,
+    Vec<String>,
+    Vec<(TokioRuntimeHandle<ClientActorInner>, Addr<ViewClientActor>, Vec<ArbiterHandle>)>,
+) {
```

Comment:
nit, though not really related to your PR: would be good to move this to a dedicated result struct